spec/controllers/accounts_controller_spec.rb

```diff
@@ -48,6 +48,7 @@
           sp_name: nil,
           decorated_user: user.decorate,
           locked_for_session: false,
+          account_reset_requested: false,
```

Comment:
it might be good to have a spec that tests with an account reset record present to make sure we construct the right kind of presenter too